lib/v2/ycwb/index.js

```diff
@@ -0,0 +1,73 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+module.exports = async (ctx) => {
+    const node = ctx.params.node ?? 1;
+    const page = ctx.params.page ?? 1;
+    const pageSize = ctx.params.pagesize ?? 15;
```

Comment:
Please use the common query parameter `ctx.query.limit` instead.
docs: https://docs.rsshub.app/parameter.html